src/solver/simulation/common-hydro-remix.cpp

```diff
@@ -263,32 +264,35 @@ namespace Simulation
 
 
 
-	bool RemixHydroForAllAreas(const Data::Study& study, PROBLEME_HEBDO& problem, uint numSpace, uint hourInYear, uint nbHour)
+	void RemixHydroForAllAreas(const Data::Study& study, PROBLEME_HEBDO& problem, uint numSpace, uint hourInYear, uint nbHour)
 	{
 		assert(nbHour == 168 && "endHour seems invalid");
 		(void) nbHour;
 		assert(study.parameters.mode != Data::stdmAdequacyDraft);
 
-		if (study.parameters.shedding.policy != Data::shpShavePeaks)
-			return true;
-
-		switch (study.parameters.simplexOptimizationRange)
+		if (study.parameters.shedding.policy == Data::shpShavePeaks)
 		{
+			bool result = true;
+
+			switch (study.parameters.simplexOptimizationRange)
+			{
 			case Data::sorWeek:
-				return Remix<  168 >(study, problem, numSpace, hourInYear);
+				result =  Remix<  168 >(study, problem, numSpace, hourInYear);
+				break;
 			case Data::sorDay:
-				return Remix<   24 >(study, problem, numSpace, hourInYear);
+				result = Remix<   24 >(study, problem, numSpace, hourInYear);
+				break;
 			case Data::sorUnknown:
 				logs.fatal() << "invalid simplex optimization range";
 				break;

```

Comment:
There was no  need for `break `because there was a `return `instead  of a use of `result `variable if  changed code.
I'm not a big fan of return in the middle of a function.

Now comment should be added in case of code change not related to the feature.